walkthroughs/eks/o11y-prometheus.md

```diff
@@ -56,6 +81,34 @@ Similarly, you can see all the scraped metrics (including application health met
 
 ## Cleanup
 
+helm3 would support --purge by default
```

Comment:
Does this line need to be here?

Reply:
Because the eks-charts repo gives outdated error hints, so give a special note here
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/123429070/220797122-58f88f56-1853-48e9-90c3-cd26ebbf91fb.png">